pkg/services/dashboards/dashboard_service.go

```diff
@@ -33,19 +33,17 @@ type DashboardProvisioningService interface {
 	DeleteProvisionedDashboard(dashboardId int64, orgId int64) error
 }
 
-// NewService factory for creating a new dashboard service
+// NewService is a factory for creating a new dashboard service.
```

Comment:
Looked at this again. I must have miss something. I cannot see that the `reqHandler` field on `dashboardServiceImpl` is being used anywhere?

Reply:
@marefr good catch, thanks. There isn't any need for it in this PR, only in the originating PR.